sympy/combinatorics/partitions.py

```diff
@@ -318,7 +318,7 @@ def __new__(cls, partition, integer=None):
         >>> IntegerPartition({1:3, 2:1})
         IntegerPartition(5, (2, 1, 1, 1))
 
-        If the value that the partion should sum to is given first, a check
+        If the value that the portion should sum to is given first, a check
```

Comment:
I'm not sure this fix is correct, please review.

Reply:
I believe these should be "partition".